collectd docs (#243)
replicatedhq/troubleshoot.sh: docs/source/collect/all.md

- [copy](./copy): copies files or folders from a pod into the collected output
- [data](./data): writes static or predefined data into the collected output
- [secret](./secret): includes information about Kubernetes secrets in the collected output

> - [collectd](./collectd): includes collectd files found on all host in the cluster

Reviewer: ```suggestion
- [collectd](./collectd): includes collectd files from all hosts in the cluster
```

> - [collectd](./collectd): includes collectd files found on all host in the cluster

Reviewer 2: ```suggestion
- [collectd](./collectd): includes collectd files found on all hosts in the cluster
```
